src/test/java/org/gridsuite/modification/server/utils/assertions/DTOAssert.java

```diff
@@ -30,9 +30,10 @@ public DTOAssert<T> recursivelyEquals(T other) {
 
     private RecursiveComparisonConfiguration getRecursiveConfiguration() {
         return RecursiveComparisonConfiguration.builder()
-            .withIgnoreAllOverriddenEquals(true)                                    // For equals test, need specific tests
-            .withIgnoredFieldsOfTypes(UUID.class, Date.class, ZonedDateTime.class)  // For these types, need specific tests (uuid from db for example)
-            .withIgnoreCollectionOrder(true)                                        // For collection order test, need specific tests
-            .build();
+                .withIgnoreAllOverriddenEquals(true)                                    // For equals test, need specific tests
+                .withIgnoredFieldsOfTypes(UUID.class, Date.class, ZonedDateTime.class)  // For these types, need specific tests (uuid from db for example)
+                .withIgnoredFields("stashed", "attachmentLine.stashed")   // Ignore the "stashed" field
```

Comment:
Remove it

Reply:
@SlimaneAmar 
when i remove this line i have almost half the tests are failing :Tests run: 433, Failures: 204, Errors: 0, Skipped: 0
, because in the tests we don't build the modification with meta data we just set the minimum number of attributes needed for the test. for example : BranchStatusModificationEnergiseSideOneLineTest.buildModificationUpdate()

with the modification i did in  AbstractNetworkModificationTest.java:170 , i successfully detect any modification where the setStashed has been forgotten.

do you want me to change those unit test also ?